terraform/README.md

```diff
@@ -0,0 +1,121 @@
+<!-- BEGIN_TF_DOCS -->
```

Comment:
maybe want to add info on updating the wowza version and the vm count if required?

Reply:
will overhaul docs once we have made other changes